www/_posts/2020-06-01-cordova-ios-release-6.0.0.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova iOS 6.0.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova iOS 6.0.0`! This is Cordova's official platform for building iOS mobile applications.
+
+* [cordova-ios@6.0.0](https://www.npmjs.com/package/cordova-ios)
+
+## Release Highlights
+
+
+**To upgrade:**
+
+```bash
+cordova platform remove ios
+cordova platform add ios@6.0.0
+```
+
+The most notable changes in this major release are:
```

Comment:
In this case, it's maybe better to hand-write this list instead of taking the commit messages.

We also maybe want to turn each item in this list into a heading below with a paragraph or two of explanation. This is a major release, there are breaking changes, we need to call out how to handle those when upgrading instead of just linking to GitHub PR comments.